templates/help.hamlet

```diff
@@ -50,6 +50,26 @@
     <p>
       Your package, together with documentation, should now appear in Pursuit.
 
+  <div #module-docs .help>
+    <h3>Writing Module Documentation
+    <p>
+      Pursuit builds documentation from module source files. Documentation is formatted
+      as [markdown](https://guides.github.com/features/mastering-markdown/) prefixed on
```

Comment:
Anyone know if pursuit supports full GFM? If not perhaps I should link somewhere else.